src/modules/dpos/validator/components/delegateProfile/delegateProfile.js

```diff
@@ -1,42 +1,91 @@
-import React, { useEffect } from 'react';
+import React, { useMemo } from 'react';
 import grid from 'flexboxgrid/dist/flexboxgrid.css';
-
+import { useTranslation } from 'react-i18next';
+import { selectSearchParamValue } from 'src/utils/searchParams';
+import { useCurrentAccount } from 'src/modules/account/hooks';
+import WalletVisualWithAddress from 'src/modules/wallet/components/walletVisualWithAddress';
+import { useBlocks } from 'src/modules/block/hooks/queries/useBlocks';
+import DialogLink from 'src/theme/dialog/link';
+import Heading from 'src/modules/common/components/Heading';
+import { PrimaryButton, SecondaryButton } from 'src/theme/buttons';
 import Box from '@theme/box';
 import styles from './delegateProfile.css';
 import DetailsView from './detailsView';
 import PerformanceView from './performanceView';
 import DelegateVotesView from './delegateVotesView';
+import { useDelegates, useSentVotes } from '../../hooks/queries';
+
+// eslint-disable-next-line max-statements
+const DelegateProfile = ({ history }) => {
```

Comment:
If you extract the SendVotes part, the resulting 2 components would be easier to read, and won't exceed the max statements rule.